layouts/_shortcodes/marketing/release-post-scheduling.html

```
@@ -0,0 +1,106 @@
+                      {{- if eq . "--"}}
+                      <a href="https://gitlab.slack.com/archives/C3TRESYPJ" title="Ping @social on Slack" target="slack"><img src="/images/slack-mark.png" style="padding:4px" alt="Slack Logo" class="avatar" height="40" width="40"> @social on Slack</a>
+                    {{- else if not (eq . "tbd") }}
+                      <a href="https://gitlab.com/{{ . }}">
+                      <img class="avatar" src="https://about.gitlab.com/images/gitlab-logo-extra-whitespace.png" width="90" height="90" alt="{{ . }}">@{{.}}
+                    </a>
+                    {{- end }}
+                  {{- end }}
+                {{- end }}
+
+              </td>
+              <td>
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </td> ], start tag match failed [ <td> ] on line 88.

(tag-pair)